tests/scripts/all.sh

```diff
@@ -2205,6 +2205,74 @@ component_test_psa_crypto_config_accel_aead () {
     make test
 }
 
+component_test_psa_crypto_config_accel_pake () {
+    msg "test: MBEDTLS_PSA_CRYPTO_CONFIG with accelerated PAKE"
+
+    # Start with full
+    scripts/config.py full
+
+    # Disable ALG_STREAM_CIPHER and ALG_ECB_NO_PADDING to avoid having
+    # partial support for cipher operations in the driver test library.
+    scripts/config.py -f include/psa/crypto_config.h unset PSA_WANT_ALG_STREAM_CIPHER
+    scripts/config.py -f include/psa/crypto_config.h unset PSA_WANT_ALG_ECB_NO_PADDING
+
+    loc_accel_list="ALG_JPAKE ALG_TLS12_PSK_TO_MS"
+    loc_accel_flags=$( echo "$loc_accel_list" | sed 's/[^ ]* */-DLIBTESTDRIVER1_MBEDTLS_PSA_ACCEL_&/g' )
+    make -C tests libtestdriver1.a CFLAGS="$ASAN_CFLAGS $loc_accel_flags" LDFLAGS="$ASAN_CFLAGS"
+
+    scripts/config.py set MBEDTLS_PSA_CRYPTO_DRIVERS
+    scripts/config.py set MBEDTLS_PSA_CRYPTO_CONFIG
+
+    scripts/config.py unset MBEDTLS_ECJPAKE_C
+
+    loc_accel_list="ALG_JPAKE"
```

Comment:
This setup is also probably wrong. 
`ALG_TLS12_PSK_TO_MS` alg is used in `ecjpake_rounds` tests. Without this line `psa_key_derivation_setup_kdf()` fails because this alg is unsupported by means of `is_kdf_alg_supported()`.

When `MBEDTLS_PSA_ACCEL_ALG_TLS12_PSK_TO_MS` is defined, then `MBEDTLS_PSA_BUILTIN_ALG_TLS12_PSK_TO_MS` is undefined:
https://github.com/Mbed-TLS/mbedtls/blob/69ae1ee4feaff9bc7e3a98ac019bde8096a9a14d/include/mbedtls/config_psa.h#L234-L238

On the other hand if I remove `ALG_TLS12_PSK_TO_MS` also from driver build `libtestdriver1_psa_key_derivation_input_internal()` fails because `LIBTESTDRIVER1_MBEDTLS_PSA_BUILTIN_ALG_TLS12_PSK_TO_MS` is undefined.

Reply:
I have removed this `ALG_TLS12_PSK_TO_MS` and disabled these tests as these tests are also failing with different components to check if CI will pass (https://github.com/Mbed-TLS/mbedtls/pull/6567/commits/31636e834ebfeedc82eaa5d3027c0f785fa2beb7, https://github.com/Mbed-TLS/mbedtls/pull/6567/commits/7e7fd266e6806765274554ad7211a175d2e0bec1).

Reply:
Driver builds have two copies of the library: the main part that's calling the driver, and a copy of the library that's acting as the driver. The build scripts create the core configuration with `ACCEL_xxx` for the mechanisms that should be accelerated, and `include/test/drivers/crypto_config_test_driver_extension.h` translates that into `BUILTIN_xxx` so that the driver implements those algorithms. This header only does the translation for some algorithms at the moment. I think it needs to do the translation for all of them. I don't understand why some algorithms are hard-coded with `#define MBEDTLS_PSA_ACCEL_xxx` here. @ronald-cron-arm ?

Reply:
Can `psa_key_derivation_input_bytes()` be called from pake driver?

![image](https://user-images.githubusercontent.com/30721012/204268107-efdac1a6-2d29-4176-ad6a-0ea962f74cac.png)